script/make_wrapper/containerd/config.toml

```diff
@@ -0,0 +1,4 @@
+[proxy_plugins]
+  [proxy_plugins.remote]
+    type = "snapshot"
+    address = "/var/lib/rs/rs.sock"
```

Comment:
nit: socks are typically located under /run

Reply:
Thank you for your review. I'll fix it on the next commit.